views/package.json

```diff
@@ -9,6 +9,6 @@
   "license": "GPL-2.0",
   "dependencies": {
     "@oat-sa/tao-test-runner": "0.8.1",
-    "@oat-sa/tao-test-runner-qti": "2.20.0"
+    "@oat-sa/tao-test-runner-qti": "github:oat-sa/tao-test-runner-qti#fix/TR-855/control-box-elements-visibility"
```

Comment:
Wrong repo name. Why do you do it manually? Just install like `npm i github:oat-sa/tao-test-runner-qti-fe#fix/TR-855/control-box-elements-visibility`